### PR TITLE
Minor updates around copyright, etc.

### LINKED ISSUE #1: Initial review
Please add comments/TODOs here for cleaning up this repository after its migration from https://github.com/joshmoore/ngff.

You can see a published version of the `main` branch under: https://ngff.openmicroscopy.org/

* [x] use orchids
* [x] Add other editors
* [x] Review copyright line (see original w3c example below)
* [x] Review title (should be added to a "how to cite" section)
* [x] Proof introduction section
* [x] Tag 0.1
* [x] Migrate content for labels, etc. and re-tag

### Original W3C copyright

<a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © [YEAR] <a href="https://www.w3.org/"><abbr title="World Wide Web Consortium">W3C</abbr></a><sup>®</sup> (<a href="https://www.csail.mit.edu/"><abbr title="Massachusetts Institute of Technology">MIT</abbr></a>, <a href="https://www.ercim.eu/"><abbr title="European Research Consortium for Informatics and Mathematics">ERCIM</abbr></a>, <a href="https://www.keio.ac.jp/">Keio</a>, <a href="https://ev.buaa.edu.cn/">Beihang</a>). W3C <a href="http://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.

### PR DESCRIPTION
**Pull request by joshmoore**
Few minor cosmetic updates around the copyright info. fix #1

### CODE REVIEW

**Comment by joshmoore** [comment]
Review:
 - http://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/joshmoore/ngff/fix-1/latest/index.bs
 - Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fngff.openmicroscopy.org%2Flatest%2F&doc2=http%3A%2F%2Fapi.csswg.org%2Fbikeshed%2F%3Furl%3Dhttps%3A%2F%2Fraw.githubusercontent.com%2Fjoshmoore%2Fngff%2Ffix-1%2Flatest%2Findex.bs

File: README.md [hunk 1]
```diff
@@ -23,6 +23,16 @@ Commits on GitHub can be viewed using web services from the W3C:
...
 
+## New versions
+
+* Make new changes to `latest/index.bs`
+* Update changelog at the bottom of `latest/index.bs`
+* Find references to previous version and _in most cases_, bump to the current version.
+* On completion:
+  - copy `latest/index.bs` to `$VERSION/index.bs`
+  - update the head matter with the new release version
+  - copy `latest/copyright.include` to `$VERSION/copyright.include`
```
Comment (sbesson) [inline README.md:34]: I assume we'll need to add a bit about the GitHub actions update so that the new version gets published

File: README.md [hunk 2]
```diff
@@ -23,6 +23,16 @@ Commits on GitHub can be viewed using web services from the W3C:
  * Rendered page: http://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/ome/ngff/master/0.2/index.bs
  * Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fngff.openmicroscopy.org%2F0.2%2F&doc2=http%3A%2F%2Fapi.csswg.org%2Fbikeshed%2F%3Furl%3Dhttps%3A%2F%2Fraw.githubusercontent.com%2Fome%2Fngff%2Fmaster%2Flatest%2Findex.bs
 
+## New versions
+
+* Make new changes to `latest/index.bs`
+* Update changelog at the bottom of `latest/index.bs`
+* Find references to previous version and _in most cases_, bump to the current version.
+* On completion:
```
Comment (sbesson) [inline README.md:31]: Is there also an action around changing the status from `Editor's draft` to `Final Community report`?

**Review by sbesson — APPROVED** [review]
A few questions around the release process but if the scope of this PR is copyright and authorship, happy to get this in

**Comment by joshmoore** [comment]
Agreed, @sbesson. Changes pushed.

**Comment by sbesson** [comment]
The CI build is failing with

```
  $ bikeshed spec "0.4/index.bs" "0.4/index.out.html"
  FATAL ERROR: Couldn't find the input file at the specified location '/home/runner/work/ngff/ngff/0.4/index.bs'.
   ✘  Did not generate, due to fatal errors
```

Is that a subtle hint that is might be time to copy over the specification under the `0.4` folder as per these instructions? /cc @constantinpape

**Comment by constantinpape** [comment]
> Is that a subtle hint that is might be time to copy over the specification under the `0.4` folder as per these instructions? /cc @constantinpape

Sounds good, let's go for it ;).

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - uses: joshmoore/spec-prod@0_3
+      - uses: ome/spec-prod@main
```
Comment (sbesson) [inline .github/workflows/publish.yml:18]: @joshmoore should we tag https://github.com/ome/spec-prod/commit/1bb5c6816c62fbe2d73ade7f2bea9affaee76c54 e.g. as `v3` to get this passing and merged.

We can then bump back to the HEAD of the spec-prod branch when we copy the content under the `0.4` folder.

Comment (joshmoore) [reply .github/workflows/publish.yml]: Done.

**Pull request merged**